test/features/parsoid/transform.js

```diff
@@ -15,123 +16,108 @@ describe('transform api', function() {
     this.timeout(20000);
     let contentTypes;
     const server = new Server();
+    const client = new REST('');
```

Comment:
Nothing to do here for you, but it would've been sweet if we could inject the config into the clients @clarakosi

Reply:
So the current workaround for the JSON file would be to declare an environment variable for the base URI. Probably easiest to do it in the test command in package.json but that would complicate things when trying to run a single file. Will look into why we did it this way next week